overcooked_ai_py/agents/benchmarking.py

```diff
@@ -20,46 +20,71 @@ class AgentEvaluator(object):
     pickleable. We should think about possible improvements/what makes most sense to do here.
     """
 
-    def __init__(self, mdp_params, env_params={}, mdp_fn_params=None, force_compute=False, mlp_params=NO_COUNTERS_PARAMS, debug=False):
+    def __init__(self, mdp_params_lst, env_params={}, mdp_fn_params_lst=None, force_compute=False, mlp_params_lst=None, debug=False):
```

Comment:
Just checking, because it's not obvious to me – although I might be missing something: is there a reason here that we need mdp_params_lst? 

Couldn't we have just have 1 mdp_params dictionary, which has a mdp generator function that returns a different mdp at each reset?

Having a list everywhere makes things quite a bit more redundant notation-wise for all the cases in which one deals with a single layout (which is probably most of the time).

Moreover, if you're planning on training on infinitely many randomly generated layouts, you won't be able to enumerate them here in any case?

Reply:
Just for some more context: my reasoning behind this reticence to modify this signature is that, because the Env class is so widely used across the codebase, it seems good to try and change it as little as possible, unless there's a strong case for why the change is absolutely necessary here or is clearly stylistically superior and will result in less bugs and cleaner code in the future.

Reply:
the reason we have a list here is because we don't necessarily have to train on environment that is coming from the same exact distribution (coming out of the same set of mdp_params). Having a list enable us to train the same agent simultaneously on two very different layout (in terms of both layout and distribution of objects)

And for the second question: we don't have to enumerate over all randomly generated layouts if we want to train on infinitely many of them because once the layout_generator works properly, we can generate it from a parameters directly instead of using "layout_name"s